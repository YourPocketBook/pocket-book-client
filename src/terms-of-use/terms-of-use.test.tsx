import { render } from "@testing-library/react";
import React from "react";
import ReactDOM from "react-dom";
import { TermsOfUse } from ".";

describe("Terms Of Use Page", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<TermsOfUse />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders correctly", () => {
    const { container } = render(<TermsOfUse />);

    expect(container).toMatchSnapshot();
  });
});
