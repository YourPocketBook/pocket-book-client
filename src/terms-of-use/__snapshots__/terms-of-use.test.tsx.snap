// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terms Of Use Page renders correctly 1`] = `
<div>
  <div
    class="background"
  >
    <div
      class="centreBox"
    >
      <h1
        class="header1"
      >
        Usage Terms and Conditions
      </h1>
      <p>
        Welcome to Pocket Book. If you continue to browse and use this website, you are agreeing to comply with and be bound by the following terms and conditions of use, which together with our privacy policy, govern our relationship with you in relation to this website. If you disagree with any part of these terms and conditions, please do not use our website.
      </p>
      <p>
        The terms "I", "me", "us", "we" and "our" all refer to the owner of this website, who is Tony Richards. The term "you" refers to the user or viewer of our website.
      </p>
      <p>
        The use of this website is subject to the following terms of use:
      </p>
      <ul>
        <li>
          The content of the pages of this website is for your general information and use only. It is subject to change without notice.
        </li>
        <li>
          This website uses cookies in accordance with our privacy policy.
        </li>
        <li>
          Neither we, nor any third party provide any warranty or guarantee as to the accuracy, timeliness, performance, completeness or suitablility of the information and materials found or offered on this website for any particular purpose. You acknowledge that such information and materials may contain inaccuracies or errors and we expressly exclude liability for any such inaccuracies or errors to the fullest extent permitted by law.
        </li>
        <li>
          Your use of any information or materials on this website is entirely at your own risk, for which we shall not be liable. It shall be your own responsibility to ensure that any products, services of information available through this website meet your specific requirements.
        </li>
        <li>
          This website contains material which is owned or licensed to us. This material includes, but is not limited to, the design, layout, look, appearance and graphics. Reproduction is prohibited other than in accordance with the copyright notice, which forms part of these terms and conditions.
        </li>
        <li>
          All trademarks reproduced in this website, which are not the property of, or licensed to the operator, are acknowledged on the website.
        </li>
        <li>
          Unauthorised use of this website may give rise to a claim for damages and/or be a criminal offence.
        </li>
        <li>
          From time to time, this website may also include links to other websites. These links are provided for your convenience to provide further information. They do not signify that we endorse the website(s). We have no responsibility for the content of the linked website(s).
        </li>
        <li>
          Your use of this website and any dispute arising out of such use of the website is subject to the laws of England, Northern Ireland, Scotland and Wales.
        </li>
      </ul>
      <p>
        This is not an official St John Ambulance website. The information enclosed is provided in good faith to volunteers from St John Ambulance, to assist them in their work. While every effort is made to ensure that the guidelines in this website are in line with current St John Ambulance policy, it is the responsibility of the user to ensure that they are following correct and current St John Ambulance policy and procedure.
      </p>
    </div>
  </div>
</div>
`;
